tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -1,101 +1,90 @@
 import * as React from 'react';
-import { shallow } from 'enzyme';
+import { waitFor } from '@testing-library/react';
 
 import {
-  createInternalReview,
   fetchUserReviews,
   setUserReviews,
   FETCH_REVIEWS,
+  FETCH_USER_REVIEWS,
```

Comment:
tbh I wasn't really clear the difference between these two constants (references to `FETCH_REVIEWS` was added to these tests by a [recent commit](https://github.com/mozilla/addons-frontend/commit/7507480ff9fe0719c3e312fd55f1a671cbefb089)) but I figured as we were only testing in the negative (that they weren't emitted) it was safe to have both 🤷